tests/ui/apps/Alert/Alert-View.js

```diff
@@ -0,0 +1,62 @@
+import Alert from '../../../../Alert';
+import Button from '../../../../Button';
+import ThemeDecorator from '../../../../ThemeDecorator';
+import spotlight from '@enact/spotlight';
+import {Component} from 'react';
+import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
```

Comment:
```suggestion
	import spotlight from '@enact/spotlight';
	import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
	import {Component} from 'react';
	
	import Alert from '../../../../Alert';
	import Button from '../../../../Button';
	import ThemeDecorator from '../../../../ThemeDecorator';
`````
Please sort by package name.